scraper/spiders/cityhall.py

```diff
@@ -76,7 +76,9 @@ def _parse_descriptions(self, raw_descriptions):
         descriptions = []
         for raw_description in raw_descriptions:
             description = raw_description.xpath(".//text()").extract()
-            document_url = raw_description.xpath(".//@href").extract_first()
+            document_url = raw_description.xpath(
+                './/a[contains(@href, "pdf")]/@href'
```

Comment:
```suggestion
                ".//a[contains(@href, 'pdf')]/@href"
```
Uma pena mudança, dá até pra aceitar pela interface do Github (só pra gente manter o padrão, como aqui:
https://github.com/DadosAbertosDeFeira/maria-quiteria/blob/ec6867f58944f60e25113a09bb06c647359dbc56/scraper/spiders/gazette.py#L53-L55